src/components/fx/hover.js

```diff
@@ -925,15 +925,10 @@ function createHoverText(hoverData, opts, gd) {
         if(d.nameOverride !== undefined) d.name = d.nameOverride;
 
         if(d.name) {
-            // strip out our pseudo-html elements from d.name (if it exists at all)
-            name = svgTextUtils.plainText(d.name || '');
-
-            var nameLength = Math.round(d.nameLength);
-
-            if(nameLength > -1 && name.length > nameLength) {
-                if(nameLength > 3) name = name.substr(0, nameLength - 3) + '...';
-                else name = name.substr(0, nameLength);
-            }
+            name = svgTextUtils.plainText(d.name || '', {
+                len: d.nameLength,
+                allowedTags: ['br', 'sub', 'sup']
```

Comment:
From our list of pseudo-html tags:

https://github.com/plotly/plotly.js/blob/cc4597257b411688b2bbc6dbe90f8a6e78cc96c8/src/lib/svg_text_utils.js#L233-L244

I can us potentially adding `<b>`, `<i>` and `<em>`. Thoughts?

Reply:
In `webGL` texts now we do support `<b>` and `<i>`. So it might be nice to have them here.

Reply:
Ok! Done in -> https://github.com/plotly/plotly.js/pull/3443/commits/6c7b0ede2e860ca9b7e03d4a849a4dedbccf2770